mautic/mautic
# [UX] Change “Clear cache” button icon from Trash to Refresh

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Currently, the ‘Clear cache’ button is represented by a Trash can icon. While it serves its purpose, the icon could potentially be misinterpreted due to its common association with deletion functions.

The pull request proposes a shift from the Trash can icon to a Refresh symbol. This symbol is widely recognized in digital interfaces and aligns more closely with the action of clearing or refreshing data, which is the actual function of the button.

This subtle yet impactful change is expected to streamline the user interface and make it more intuitive. It primarily benefits users who frequently interact with the ‘Clear cache’ button, as it reduces ambiguity and enhances the clarity of the button’s function.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Marketplace
3. See the new icon!

OBS.: Merging the new icon pack is necessary. We may have to perform a rebase after completing the icon pack merge before this PR takes effect.
